specification/web/resource-manager/Microsoft.Web/2016-03-01/Diagnostics.json

```diff
@@ -0,0 +1,1532 @@
+                "readOnly": true
+              },
+              "description": {
+                "description": "Description of the detector",
+                "type": "string",
+                "readOnly": true
+              },
+              "rank": {
+                "format": "double",
+                "description": "Detector Rank",
+                "type": "number",
+                "readOnly": true
+              },
+              "isEnabled": {
+                "description": "Flag representing whether detector is enabled or not.",
+                "type": "boolean",
+                "readOnly": true
+              }
+            },
+            "x-ms-client-flatten": true
+          }
+        }
+      },
+      "DiagnosticAnalysis": {
+        "description": "Class representing a diagnostic analysis done on an application",
+        "type": "object",
+        "properties": {
+          "startTime": {
+            "format": "date-time",
+            "description": "Start time of the period",
+            "type": "string"
+          },
+          "endTime": {
+            "format": "date-time",
+            "description": "End time of the period",
+            "type": "string"
+          },
+          "abnormalTimePeriods": {
+            "description": "List of time periods.",
+            "type": "array",
+            "items": {
+              "$ref": "#/definitions/AbnormalTimePeriod"
+            }
+          },
+          "payload": {
+            "description": "Data by each detector",
+            "type": "array",
+            "items": {
+              "$ref": "#/definitions/AnalysisData"
+            }
+          },
+          "nonCorrelatedDetectors": {
+            "description": "Data by each detector for detectors that did not corelate",
+            "type": "array",
+            "items": {
+              "$ref": "#/definitions/DetectorDefinition"
+            }
+          }
+        }
+      },
+      "DiagnosticAnalysisCollection": {
+        "description": "Collection of Diagnostic Analyses",
+        "required": [
+          "value"
+        ],
+        "type": "object",
+        "properties": {
+          "value": {
+            "description": "Collection of resources.",
+            "type": "array",
+            "items": {
+              "$ref": "#/definitions/AnalysisDefinition"
+            }
+          },
+          "nextLink": {
+            "description": "Link to next page of resources.",
+            "type": "string"
+          }
+        }
+      },
+      "DiagnosticCategory": {
+        "description": "Class representing detector definition",
+        "type": "object",
+        "allOf": [
+          {
+            "$ref": "./CommonDefinitions.json#/definitions/ProxyOnlyResource"
+          }
+        ],
+        "properties": {
+          "properties": {
+            "description": "DiagnosticCategory resource specific properties",
+            "properties": {
+              "diagnosticCategoryName": {
```

Comment:
How is this property different from the name  property which is described in the ProxyOnlyResource model? we try and avoid duplicate properties, not sure if its the case here.

Reply:
I can remove this and use the main name property